crates/store/re_types/src/datatypes/tensor_data_ext.rs

```diff
@@ -311,13 +320,18 @@ impl<D: ::ndarray::Dimension> TryFrom<::ndarray::Array<half::f16, D>> for Tensor
             })
             .collect();
         if value.is_standard_layout() {
+            let (vec, offset) = value.into_raw_vec_and_offset();
+            // into_raw_vec_and_offset() guarantees that the logical element order (.iter()) matches the internal
+            // storage order in the returned vector if the array is in standard layout.
+            let vec_slice = if let Some(offset) = offset {
+                &vec[offset..]
+            } else {
+                debug_assert!(vec.is_empty());
```

Comment:
Why is the vector expected to be empty here?

Reply:
into_raw_vec_and_offset() returns `None` when the Array is empty. The ndarray 0.16.1 documentation incorrectly states that it returns 0 in this case (I fixed that [here](https://github.com/rust-ndarray/ndarray/pull/1432))